templates/bricks/overview.html.twig

```diff
@@ -1,11 +1,19 @@
+{% set hasVersions = bricks|filter(b => b.version is not empty)|length > 0 %}
+{% set hasDescriptions = bricks|filter(b => b.description is not empty)|length > 0 %}
```

Comment:
Cause these variable declarations are nearly the same and will be used only 1(!) time in code I would have implemented Twig functions for them.
Something like:
```
   {% if (has(bricks, 'version') %}
      ...
```

Reply:
I added the `any()` filter to twig, which makes it at least a bit shorter (and more performant for the truthy case).

Note that the filter should better be a test, but arrow functions in tests will only be allowed in Twig `>=3.15`, which is not released yet (and we need to support Twig `^3.8` because of Pimcore 10).